src/components/CalloutBanner.tsx

```diff
@@ -90,7 +91,7 @@ const CalloutBanner: React.FC<IProps> = ({
     <Image
       image={image}
       alt={alt}
-      maxImageWidth={maxImageWidth}
+      maximagewidth={maxImageWidth}
```

Comment:
I was getting a warning about this prop name and it suggested me to lowercase it.